### PR TITLE
Improve API documentation

### PR DESCRIPTION
**Pull request by nbarnabee**
This addresses [T330126](https://phabricator.wikimedia.org/T330126).

It has been branched from #31

### CODE REVIEW

File: api/routes.py [hunk 1]
```diff
@@ -84,49 +84,49 @@ def get(self, query_args):
...
-tasks = Blueprint("tasks", __name__, description="Fetching and updating Toolhunt tasks")
+tasks = Blueprint("tasks", __name__, description="Get incomplete tasks and submit data to Toolhub.")
 
 
 @tasks.route("/api/tasks")
 class TaskList(MethodView):
     @tasks.response(200, TaskSchema(many=True))
     def get(self):
-        "Return a bundle of 10 incomplete tasks."
+        "Get a bundle of ten incomplete tasks."
```
Comment (blancadesal) [inline api/routes.py:114]: "Get ten incomplete tasks" would be shorter and conveys the same information.

File: api/routes.py [hunk 2]
```diff
@@ -84,49 +84,49 @@ def get(self, query_args):
...
-        "Return information about a specific task."
+        "Get information about a specific task."
         task = Task.query.get_or_404(task_id)
         return task
 
     @tasks.arguments(TaskCompleteSchema)
     @tasks.response(201)
     def put(self, task_data, task_id):
-        """Update a tool record on Toolhub."""
+        """Update a record on Toolhub and in our database."""
```
Comment (blancadesal) [inline api/routes.py:129]: I prefer the previous version – the fact that we update the database is an implementation detail on the backend. Imo, "tool" is more descriptive than "record" too.

File: api/routes.py [hunk 3]
```diff
@@ -159,15 +159,15 @@ def put(self, task_data, task_id):
 
 
 user = Blueprint(
-    "user", __name__, description="Get information about the currently logged-in user."
```
Comment (blancadesal) [inline api/routes.py:162]: "currently logged-in" is more descriptive and precise than just "current", imo.

File: api/routes.py [hunk 4]
```diff
@@ -159,15 +159,15 @@ def put(self, task_data, task_id):
...
+    "user", __name__, description="Get information about the current user."
 )
 
 
 @user.route("/api/user")
 class CurrentUser(MethodView):
     @tasks.response(200, UserSchema)
     def get(self):
-        """Get the username of currently logged-in user."""
+        """Get the Mediawiki username of the current user."""
```
Comment (blancadesal) [inline api/routes.py:170]: Idem, prefer the previous version.

**Review by blancadesal — COMMENTED** [review]
I've left some comments, some of which come down to personal preference, so feel free to ignore.

**Comment by nbarnabee** [comment]
> I've left some comments, some of which come down to personal preference, so feel free to ignore.

Nah, I agree with your comments.  I struggle to strike a good balance between length and clarity.  Missing that one period was just embarrassing.

**Pull request merged**